CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+0.5.0 (2021-06-28)
+===
+
+* Updates to the rebased JikesRVM.
```

Comment:
Probably use the word 'upstream' or 'latest' instead of 'rebased'. You can also be clear which rev or version it is.

Reply:
Done